core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -156,9 +189,20 @@ export class ResultPanelComponent {
       this.displayErrorMessage(`execution doesn't have any results`);
       return;
     }
-
-    // execution success, display result table
-    this.displayResultTable(response);
+    this.executeWorkflowService.updateResultMap(response);
+    // If the selected operator is a sink operator, the result panel should display result when response is coming.
+    if (this.selectedOperatorID !== '') {
```

Comment:
Continue with the `selectedOperatorID` point above. This is currently the only place where we need `selectedOperatorID`, but we can just call `this.workflowActionService.getJointGraphWrapper().getCurrentHighlightedOperatorIDs()` to retrieve the information.

Reply:
Yes, I will remove `this.selectedOperatorID` from consistency and simplicity.